bin/resources/GameIndex.yaml

```diff
@@ -32254,21 +32263,36 @@ SLPS-20165:
 SLPS-20167:
   name: "La Pucelle"
   region: "NTSC-J"
+SLPS-20168:
+  name: "NHL 2002"
+  region: "NTSC-J"
 SLPS-20169:
   name: "Chou! Rakushii Internet Tomodachi Nowa [Type-PG]"
   region: "NTSC-J"
+SLPS-20170:
+  name: "Wave Rally"
+  region: "NTSC-J"
 SLPS-20171:
   name: "Smash Court Tennis Pro"
   region: "NTSC-J"
+SLPS-20173:
+  name: "Magical Sports - Hard Hitter 2"
```

Comment:
might be worth putting "Hard HItter 2" instead? Just thinking of people searching it in Qt? Idk, Your choice I guess..

Reply:
PAL name is just Hard Hitter 2, but Japanese name is Magical Sports - Hard Hitter 2.
https://www.jp.playstation.com/software/title/slps20173.html
Hard Hitter 1 (SLPS-20098) is also in GameDB already named as Magical Sports - Hard Hitter, so I'd have to rename that to only Hard Hitter to be consistent.

Reply:
Just thinking of easy of searching the Game list, if you're looking for Hard Hitter 2, you will type "hard" more likely.  Magical Sports seems to be the series, so hmm... IDK, I guess it's a stylistic choice, it could be either

Hard Hitter 2
Magical Sports - Hard Hitter 2
Hard Hitter 2 - Magical Sports

I'll let you decide creatively :) Personally I prefer the first one for ease of use.

Reply:
I'll keep the full Magical Sports - Hard Hitter 2 name, because that seems to be the most commonly used on Japanese websites. Also, most people searching for the game under H will probably play the PAL version anyway.
For Japanese users it's probably a nightmare to find games anyway since the games are all written in romaji in the GameDB and also lexicographic order works differently in Japanese than sorting A-Z

Reply:
Alright, I'm happy with that :)